client/collection_field_description.go

```diff
@@ -38,6 +38,9 @@ type CollectionFieldDescription struct {
 	//
 	// Otherwise will be [None].
 	RelationName immutable.Option[string]
+
+	// DefaultValue contains the default value for this field.
+	DefaultValue any
```

Comment:
todo: Please test the PatchCollection behaviour for this new property

Reply:
done